### DISCUSSION TITLE
Frequent calulation possible

### DISCUSSION THREAD

**Discussion by Linkmonster** [jeroenterheerdt/HAsmartirrigation#374] [Q&A]
Hi @jeroenterheerdt 
first of all, great work!!! Thanks for sharing.

I am currenttly setting up my watering system for my greenhouse and garden.
for curiosity, I have set up a schedule to calulate very hour (with 15 min sensor update) in order to get a feeling for the system.
However it seemed that this has a negative impact on the calulation as I ended up with rising buckets (going from 0 to +10mm) within a day (rather warm day ~20°C, low humidity and quite a bit of wind). One of my thoughts was that the expected rain in the coming days could be the cause for this (forecast was set to 3 days and a bit of rain was predicted on day +2)
Strangely also my green house zone did not work as expected, tempearture around 25-30°C, humidity 20-30% prescipitation set to fixed 0 and I accumulated 3 seconds of watering time (bucket ~ 1/1000th of a mm).

What do you think, is frequent calulation (without resetting the bucket for sure) something doable or should we stick to the 24h scheme?

Best regards
Flo

**Comment by david-j-duran** [comment]
I also have been seeing this. Assumed it should be ok since there is a service and manual option to Update and Calculate for the Zones. Could this also be why I'm seeing increasing bucket (positive) on an 85 F sunny day here in NJ, USA.
